servicetalk-concurrent/src/main/java/io/servicetalk/concurrent/BlockingIterable.java

```diff
@@ -64,6 +64,7 @@ public interface BlockingIterable<T> extends CloseableIterable<T> {
     default void forEach(Consumer<? super T> action, LongSupplier timeoutSupplier, TimeUnit unit)
             throws TimeoutException {
         requireNonNull(action);
+        // TODO: should we here try/catch with close explicitly now?
```

Comment:
@idelpivnitskiy wdyt 🤔

Reply:
Discussed offline: bcz in this case we own `Iterator` instance, we should keep it within try-with-resources block for all `forEach` overloads, even the one without timeout args.